packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/DateTimePickerOnChange.test.jsx

```diff
@@ -4,20 +4,73 @@ import DateTimePicker from '../../../DateTimePicker';
 class DatePickerOnChange extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { date: '' };
+    this.state = {
+      date: '',
+      iSO: '',
+      inputValue: '',
+      dateValue: '',
+      timeValue: '',
+      isAmbiguousHour: 'No',
+      isCompleteValue: 'No',
+      isValidValue: 'Yes',
+    };
     this.handleDateChange = this.handleDateChange.bind(this);
   }
 
-  handleDateChange(event, date) {
-    this.setState({ date });
+  handleDateChange(event, date, options) {
+    this.setState({
+      date,
+      iSO: options.iSO,
+      inputValue: options.inputValue,
+      dateValue: options.dateValue,
+      timeValue: options.timeValue,
+      isAmbiguousHour: options.isAmbiguousHour ? 'Yes' : 'No',
+      isCompleteValue: options.isCompleteValue ? 'Yes' : 'No',
+      isValidValue: options.isValidValue ? 'Yes' : 'No',
+    });
   }
 
   render() {
     return (
       <div>
         <h3>
-Selected Date:
+          Selected Date:
           <span id="selected-date">{this.state.date}</span>
+          <br />
+          <br />
+          ISO String:
+          {' '}
+          <span id="iso">{this.state.iSO}</span>
+          <br />
+          <br />
+          Input Value:
+          {' '}
+          <span id="input-value">{this.state.inputValue}</span>
+          <br />
+          <br />
+          Date Value:
+          {' '}
+          <span id="date-value">{this.state.dateValue}</span>
+          <br />
+          <br />
+          Time Value:
+          {' '}
+          <span id="time-value">{this.state.timeValue}</span>
+          <br />
+          <br />
+          Is Ambiguous?
+          {' '}
+          <span id="ambiguous-date">{this.state.isAmbiguousHour}</span>
+          <br />
+          <br />
+          Is Date Complete?
```

Comment:
nit: this should be Is-Date-Time Complete since it is validating both :)

Reply:
Addressed here : https://github.com/cerner/terra-framework/pull/1045/commits/1e8dc74b082dc0de2c1002be51b425e1d49f9784